rest-assured-tests/src/test/java/tariffManagementService/TariffObject.java

```diff
@@ -0,0 +1,19 @@
+package tariffManagementService;
+
+import com.google.gson.Gson;
+import io.restassured.response.Response;
+
+class TariffObject {
+    int id;
```

Comment:
class can be public I guess
private on fields
@Getter/Setter on class

Reply:
done